src/components/NcDialog/NcDialog.vue

```diff
@@ -113,8 +114,12 @@ export default {
 		return {
 			showDialog: false,
 			newName: '',
-			currentName: 'none yet.',
+			currentName: 'non yet.',
```

Comment:
non?

Reply:
no idea wanted to fix this right now :sweat_smile: 

```suggestion
			currentName: 'none yet.',
```